portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,8 +36,20 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println(convertToJsonUsingGson(comments));
+    
+    final Query allComments = new Query("Comment");
+    DatastoreService dataStore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = dataStore.prepare(allComments);
+
+    ArrayList<String> commentList = new ArrayList<>();
+    for (Entity entity : results.asIterable()) {
+        final String comment = (String) entity.getProperty("comment-field");
```

Comment:
Something to think about: What would happen in this code if `entity.getProperty("comment-field")` returns a null? 
Will this cause an error, or would the code function correctly?

Reply:
When I click submit when nothing is entered it behaves as usual. Right now I'm trying to add more fields, like a name field and a date field and a timestamp, what would be the best way to add all these fields?

Reply:
Timestamp is probably something you want to generate on the server side. The simplest way is to use `System.getCurrentTimeMillis()`.